edgebuilder-install.sh

```diff
@@ -292,6 +293,7 @@ install_node()
     echo "ERROR: Node installation could not be validated"
   else
     echo "INFO: Node validation succeeded"
+    echo $OUTPUT
```

Comment:
The formatting of this output looks pretty bad. Why don't we just run the edgebuilder-node command and it will automatically print it's help all nicely formatted?

Reply:
ah my bad. I did that originally and thought i might as well make use of output. I didnt test the change thinking the output would be the same. I will change it back

Reply:
changes made